src/Migrators/Migrators.MSSQL/Migrations/Application/20211018230457_InitialCreate.cs

```diff
@@ -151,6 +161,12 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         principalTable: "Roles",
                         principalColumn: "Id",
                         onDelete: ReferentialAction.Cascade);
+                    table.ForeignKey(
```

Comment:
Possibly an unintentional duplication of 'FK_RoleClaims_Roles_RoleId'?

Reply:
Yep... the same thing here